WikiWatershed/model-my-watershed
# Logos In Splash Page and Fix Compare Scroll

## Overview

- Move logos out of header and into splash page
- Allow the compare view to scroll on short viewports

Connects #1869 
Connects #2027 

### Demo

<img width="843" alt="screen shot 2017-07-12 at 1 51 45 pm" src="https://user-images.githubusercontent.com/7633670/28131953-75295cce-6709-11e7-8912-594d0aca97c1.png">

![djrzsz3i9n](https://user-images.githubusercontent.com/7633670/28131974-84db7544-6709-11e7-911e-637929990147.gif)

## Testing Instructions

 * `./scripts/bundle.sh --vendor` `./scripts/bundle.sh`
 * Look at the splash page and confirm the logos look nice and work as links
 * Go to the compare view on tall and narrow viewports confirming it scrolls where appropriate
